### DISCUSSION TITLE
Cannot build site

### DISCUSSION THREAD

**Discussion by lakshyapriyadarshi** [razonyang/hugo-theme-bootstrap#448] [Q&A] [answered]
I am having the following error while trying to install the theme:

![Screenshot 2022-03-18 15:20:25](https://user-images.githubusercontent.com/37311397/158981252-11e635f6-342f-4cb0-ad40-1f6a6f321e96.png)

Kindly help me with identifying if I am missing something?

Thanks

**Comment by razonyang** [comment, marked as answer]
Thanks for using this theme. You'll need to install the CSS/JS dependencies via `npm`.

```shell
$ hugo mod npm pack
$ npm install
$ hugo server
```

Please add the following configuration to `config.toml`, because this theme need this to be set up, so that we can purge the unused CSS.

```toml
// config/_default/config.toml
[build]
  writeStats = true
```

**Reply by lakshyapriyadarshi** [reply]
Kindly excuse my ignorance and thanks for the help. I am now able to build the site. 

However, the page is rendered as follows:

![Screenshot 2022-03-18 15:54:17](https://user-images.githubusercontent.com/37311397/158986426-11b98547-33cb-4f81-abcb-bf495a980d2e.png)

When I tried building it by cloning the repository and following the instructions, I got the same rendered page. 

Am I still missing something?

**Reply by lakshyapriyadarshi** [reply]
Sorry, I figured the mistake, and now the page builds as it should. 

Thanks a lot for the help.

**Reply by razonyang** [reply]
My pleasure.
